### DISCUSSION TITLE
When adding a button, how do I designate which rendering window to show it?

### DISCUSSION THREAD

**Discussion by DYT-Flyer** [marcomusy/vedo#458] [Q&A] [answered]
A couple of other questions:
Is there a way to prevent the user from min/max/stretching the window when played?
Is there a way to define preference for overlap (button1 appears over button2)?

**Comment by marcomusy** [comment, marked as answer]
1. what do you mean "when palyed"?
2. I guess by the order of adding them to the rendering window... but I never tested it..

**Reply by DYT-Flyer** [reply]
1. Sorry, I was referring to when Plotter.show() is called.
2. This works, I didn't know if there is a more advanced option.

**Reply by marcomusy** [reply]
1. oh, no there isn't, that's owned by the operative system
2. ok, no , the only way is that i guess..

**Reply by DYT-Flyer** [reply]
Is there a way to define the size of the buttons?

**Reply by marcomusy** [reply]
Yes, e.g.: `plotter.addButton(buttonfunc, states=[" Play ","Pause"], size=90)`
try `vedo --search button`
